root/tmva/tmva/ConvNetBenchmarks.h

```diff
@@ -0,0 +1,105 @@
+#include "TMVA/Factory.h"
```

Comment:
As far as I can see there are no template definitions in this header. Then the implementation is better put in the implementation file (.cxx)